### DISCUSSION TITLE
Make all 4 ruleset more playable in mobile

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#19304] [Ideas] [answered]
Currently I can't really play osu! on pc and decided to play osu!lazer on mobile and decided to test out mania and standard but HOLY HELL for standard the circles are too small and you can't alternate, for mania the are of hitting notes are too small for any skin, please make an update on this.

**Comment by ghost** [comment]
Also do note, I played standard recently and mania since mania touch was introduced

**Comment by zachmanthethird** [comment, marked as answer]
Currently tracking via #18882 :+1:
